templates/BreadcrumbsTemplate.ss

```diff
@@ -1,6 +1,10 @@
-<% if $Pages %>
-    <nav class="breadcrumbs" aria-label="Breadcrumbs">
-        <ol class="breadcrumbs__list">
+<nav class="breadcrumbs" aria-label="Breadcrumbs">
+    <ol class="breadcrumbs__list">
+        <li class="breadcrumbs__list-item">
+            <a href="$BaseURL">Home</a>
+            <span class="breadcrumbs__list-delimiter" aria-hidden="true">/</span>
+        </li>
+        <% if $Pages %>
             <% loop $Pages %>
                 <% if not $IsLast %>
```

Comment:
```suggestion
                <% if not $IsLast && not $isHomePage %>
```
This is one option, see  for the other option.
Only do _one_ of these.

This option means the home page breadcrumb always says "Home" even for direct children of the home page (as opposed to root level pages) regardless of what the menu title is for the home page.

Reply:
This option is in line with what James had been pushing for so lets do this one :)